ReplicationAcceptance/ReplicationAcceptance/ReplicationAcceptance.m

```diff
@@ -128,7 +148,9 @@ -(CDTReplicator *) pullFromRemoteWithFilter:(NSString*)filterName params:(NSDict
         [NSThread sleepForTimeInterval:1.0f];
         NSLog(@" -> %@", [CDTReplicator stringForReplicatorState:replicator.state]);
     }
-    
+
+    XCTAssertGreaterThanOrEqual(interceptor.timesRequestIntercepted, 1);
+
```

Comment:
re-run replication after clearing the interceptors and check the count hasn't gone up?